addons/osio-addon/src/main/java/io/fabric8/launcher/osio/client/api/Namespace.java

```diff
@@ -0,0 +1,31 @@
+package io.fabric8.launcher.osio.client.api;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonDeserialize(as = ImmutableNamespace.class)
+@JsonSerialize(as = ImmutableNamespace.class)
+public interface Namespace {
+
```

Comment:
Namespace is used only in Tenant. Why move to a top level class?

Reply:
Yes